scvi/dataset/anndataset.py

```diff
@@ -0,0 +1,141 @@
+import logging
```

Comment:
Would using `git rename anndata.py anndataset.py` maintain the git history of the file even though name has changed?

Reply:
Do you think it could be a GitHub glitch? Because the PyCharm git blame still has the git history in the new `anndataset.py`. I am not familiar with that.

Reply:
That could be it. If not, no big deal.